mesheryctl/internal/cli/root/perf/list.go

```diff
@@ -0,0 +1,212 @@
+package perf
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/inancgumus/screen"
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/layer5io/meshery/models"
+	"github.com/pkg/errors"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	page         uint
+	totalPage    uint
+	totalResults uint
+	limitResults uint = 10
+)
+
+var listCmd = &cobra.Command{
+	Use:     "list",
+	Short:   "List Performance profiles",
+	Long:    `List all the available performance profiles`,
```

Comment:
Shouldn't we also add about listing the results within a performance profile?

Reply:
@navendu-pottekkat I added the names with IDs, and I am also not sure about clearing the screen thing, if that's not a good user experience then we have to limit results up to 25 at max.

Reply:
Can how you recorded this?

Reply:
Potential inspiration - https://en.wikipedia.org/wiki/More_(command)#/media/File:Unix_more_output.png

More (%)

Spacebar to advance. "q" or "ctrl+c" to stop, perhaps.

Reply:
you mean I have to implement a keypress listener?

Reply:
Yes, indeedy.